jvm/workbookapp/otter.install4j

```diff
@@ -315,7 +315,7 @@ return console.askYesNo(message, true);
                     </element>
                     <element index="1">
                       <object class="java.io.File">
-                        <string>${installer:sys.userHome}/.config/Orature</string>
+                        <string>${installer:sys.userHome}${installer:sys.fileSeparator}Orature</string>
```

Comment:
This looks like I changed `/.config/` to `${installer:sys.fileSeparator}` but that's not what I did. I removed the entire `/.config/` line because it wasn't functional. That was for linux and is fixed with the new script at line 567. I then added the `<string>${installer:sys.userHome}${installer:sys.fileSeparator}Orature</string>` line for windows because there are files in two directories in windows.